app/src/androidTest/java/com/duckduckgo/app/httpsupgrade/HttpsUpgraderTest.kt

```diff
@@ -31,7 +31,7 @@ class HttpsUpgraderTest {
     @Before
     fun before() {
         mockDao = mock()
-        testee = HttpsUpgrader(mockDao)
+        testee = HttpsUpgraderImpl(mockDao)
```

Comment:
The great implementation name debate.  But nothing to change here, another one to discuss probably.